contributions/q1-25/draft.md

```diff
@@ -349,9 +349,14 @@ As a leader looking at how to optimize the AI supply chain security of your orga
 3. Create an ethics and risk committee focused specifically on AI for your organisation  
 4. Ensure a basic level of AI understanding is present within your senior stakeholders
 
+
+
 ### 4.2 Security Practitioner {#4.2-security-practitioner}
 
-In the realm of AI security, security practitioners are a critical piece in ensuring that these advanced technologies are secure and reliable. Their role is essential, as they are tasked with implementing safeguards at every stage of the AI lifecycle, from development to deployment and beyond. While they are not directly involved in creating the models, their expertise is essential in building a robust security framework around how models are developed and consumed, 
+In the realm of AI security, security practitioners are a critical piece in ensuring that these advanced technologies are secure and reliable. Their role is essential, as they are tasked with implementing safeguards at every stage of the AI lifecycle, from development to deployment and beyond. While they are not directly involved in creating the models, their expertise is essential in building a robust security framework around how models are developed and consumed.
+
+**Note on AI Defensive Frameworks: **
+_For additional insights of the existing frameworks that security practitioners can use to protect their AI systems, please refer to CoSAI’s “Preparing Defenders for a Changing Security Landscape" workstream. This workstream provides resources that can serve as a comprehensive review of current defender frameworks (including NIST, MITRE, OWASP, and others) and details a gap analysis that complements the risk and control assessment presented in this paper._
```

Comment:
This seems like it should be at the end of 2.3 where we highlight existing frameworks and contrast a few.

Reply:
Great point! I will move it up now.